Firebase/Auth/Source/FIRAuth.m

```diff
@@ -19,7 +19,7 @@
 #import "FIRAuth_Internal.h"
 
 #import "FIRAppAssociationRegistration.h"
-#import "FIRAppInternal.h"
+#import <FirebaseCore/FIRAppInternal.h>
```

Comment:
Nit: Doesn't look like the other headers are ordered, maybe we should do a full pass at the `import` statements to sort them all consistently.

Reply:
Thanks.

Some components use style and some don't - and some it breaks like Database - so let's do sorting separately for the unstyled components.